CMakeLists.txt

```diff
@@ -452,6 +468,15 @@ foreach(target devilution ${BIN_TARGET})
       -lfreetype -lvorbisfile -lvorbis -logg -lmodplug -lmpg123 -lSDL2 -lopusfile -lopus -lEGL -lglapi -ldrm_nouveau -lpng -lbz2 -lz -lnx)
   endif()
 
+  if(AMIGA)
+    target_link_libraries(${target} PRIVATE
+      ${FREETYPE_LIBRARY}
+      ${ZLIB_LIBRARY})
+    if(NOT WARPOS)
+      target_link_libraries(${target} PRIVATE -ldebug)

```

Comment:
Do we need this? Even when not building in release mode?

Reply:
Yes WarpOS is build as debug :/

Reply:
Does it crash otherwise?

Reply:
Yes afaik, @MBeijer ?